docs/4.2/architecture/teleport_architecture_overview.md

```diff
@@ -219,11 +218,11 @@ $ tsh --proxy=p scp -P example.txt user@host/destination/dir
 ```
 
 Unlike `ssh`, `tsh` is very opinionated about authentication: it always uses
-auto-expiring keys and it always connects to Teleport nodes via a proxy.
+auto-expiring certificates and it always connects to Teleport nodes via a proxy.
 
-When `tsh` logs in, the auto-expiring key is stored in `~/.tsh` and is valid for
-12 hours by default, unless you specify another interval via the `--ttl` flag
-(capped by the server-side configuration).
+When `tsh` logs in, the auto-expiring certificate is stored in `~/.tsh` and is
```

Comment:
It's good to have some fresh eyes on these docs. +1 for using certificates vs keys